ci/environment-docs.yml

```diff
@@ -0,0 +1,27 @@
+name: xmovie-docs
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - python=3.8
+  # Core
+  - xarray
+  - matplotlib
+  - cartopy
+  - ffmpeg
+  # Optional
+  - tqdm
+  - dask-core
+  # Examples
+  - pooch
+  - jupyterlab
+  - ipywidgets
+  # Docs
+  - sphinx>=4.0
+  - sphinx-autobuild
+  - sphinx-book-theme
+  - nbsphinx
+  #
+  - pip
+  - pip:
+    - '-e ../'  # xmovie
```

Comment:
I am not sure about this part here. I would prefer to add a short contributor guide, with instructions on how to set up the env and then install xmovie with `pip install -e .`, because this here might mess with RTD?

Reply:
The issue is that `xmovie` needs to be installed or at least importable since I am loading the version from it. This is a pretty foolproof method. I can confirm it is valid Conda env spec and doesn't mess with RTD.
Xarray [does the same thing](https://github.com/pydata/xarray/blob/f76ea09bebba35a175868a81845c426890ec0330/ci/requirements/doc.yml#L37-L41).
I do think contributor guide with instructions for setting up the envs for normal work and docs work would be a good addition!